test/Sandpit/Exscientia/Free_Energy/test_restraint.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+import BioSimSpace.Sandpit.Exscientia as BSS
+from BioSimSpace.Sandpit.Exscientia.Align._decouple import decouple
```

Comment:
Should it be _decouple or decouple? My understanding was that the user would have to directly use decouple, so would `from BioSimSpace.Sandpit.Exscientia.Align.decouple import decouple` be better?

Reply:
Yes, you are right, I guess I copied and pasted this command from somewhere so hadn't changed it.